Makefile.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @fileoverview Build file
+ * @author nzakas
+ */
+/* global echo, exit, find, target */
+
+"use strict";
+
+/* eslint no-console: 0*/
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+require("shelljs/make");
+
+var nodeCLI = require("shelljs-nodecli");
+
+//------------------------------------------------------------------------------
+// Data
+//------------------------------------------------------------------------------
+
+var NODE_MODULES = "./node_modules/",
+
+    // Utilities - intentional extra space at the end of each string
+    MOCHA = NODE_MODULES + "mocha/bin/_mocha ",
+
+    // Files
+    MAKEFILE = "./Makefile.js",
+    /* eslint-disable no-use-before-define */
+    JS_FILES = find("lib/").filter(fileType("js")).join(" "),
+    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" ");
+    /* eslint-enable no-use-before-define */
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Generates a function that matches files with a particular extension.
+ * @param {string} extension The file extension (i.e. "js")
+ * @returns {Function} The function to pass into a filter method.
+ * @private
+ */
+function fileType(extension) {
```

Comment:
I know you just copied it from ESLint makefile, but you can use `ls('lib/**/*.js')` instead (I'm pretty sure you can, at least)